### PR TITLE
Beets 204 database fixes, updated

### PR DESCRIPTION
**Pull request by levertyk**
### Definition of Done:

* [x] Have you checked your [Jira](https://bee-cs458.atlassian.net/jira/software/projects/BEETS/boards/2/backlog) task, to make sure the user acceptance criteria are met?
* [x] Does your code have comments describing it's function and reason?

### Tests

* [x] Do you have Jest tests?
* [x] Have you documented your testing criteria and edge cases in your Jira sub-task?

### Source Control

* [x] Is all relevant code checked-in?
* [x] Is this branch ready to be reviewed by the Tech Lead?

### Formatting

* [x] Do your variable names make sense (to others)?
* [x] Is your code style consistent with other code in the file or the [Style Guide](https://google.github.io/styleguide/jsguide.html)

### CODE REVIEW

**Comment by piratecody** [comment]
Pls update with main

File: utilities/DatabaseUtilities.js [hunk 1]
```diff
@@ -38,7 +38,9 @@ async function getConnection(extraConfig) {
     port: process.env.DB_PORT,
     database: process.env.DB_NAME,
     password: process.env.DB_PASSWORD,
-    // Removed SSL Verification for new database
+    ssl: {
+      ca: fs.readFileSync(process.env.CERT_PATH),
```
Comment (piratecody) [inline utilities/DatabaseUtilities.js:42]: SSL security is cool and good

**Review by piratecody — APPROVED** [review]
Very good work, Mr. Scrum Master!
